### ISSUE TITLE
Model collapses around 300K training steps

### ISSUE THREAD

**Issue by duchengbin8** [CompVis/latent-diffusion#357]
![Image20240331213845](https://github.com/CompVis/latent-diffusion/assets/69499665/a2907ed6-3483-4535-ac7c-0f5d914881eb)
![Image20240331213841](https://github.com/CompVis/latent-diffusion/assets/69499665/e6851d4d-74d8-4840-bd4a-49c0de94f5b5)
I am trying to train the FFHQ and Bedrooms data sets from scratch, but both models collapses around 300K training steps. The FID is steadily declining during the training process, and the loss does not appear NAN. Does anyone know what is going on?

**Comment by lazywu170101** [comment]
It happened to me.
